src/OpenCV/Core/ArrayOps.hs

```diff
@@ -609,6 +610,16 @@ matSplit src = unsafePerformIO $
 <http://docs.opencv.org/3.0-last-rst/modules/core/doc/operations_on_arrays.html#minmaxloc OpenCV Sphinx doc>
 -}
 
+{- | Apply the same 1 dimensional action to every channel
```

Comment:
You accidentally placed this function between `minMaxLoc` and its documentation.